lua/telescope/_extensions/frecency.lua

```diff
@@ -35,22 +35,16 @@ local state = {
   picker = {},
 }
 
-local function format_filepath(filename, opts)
-  local original_filename = filename
-
-  if state.active_filter then
-    filename = Path:new(filename):make_relative(state.active_filter)
-  else
-    filename = Path:new(filename):make_relative(state.cwd)
-    -- check relative to home/current
-    if vim.startswith(filename, os_home) then
-      filename = "~/" .. Path:new(filename):make_relative(os_home)
-    elseif filename ~= original_filename then
-      filename = "./" .. filename
-    end
+local function filepath_formatter(opts)
+  local path_opts = {}
+  for k, v in pairs(opts) do
+    path_opts[k] = v
```

Comment:
whats the point of this loop.

we just copy opts into path_opts arn't we?

Reply:
yeah we can do it like this

Reply:
Yes, we need to clone opts here to overwrite later if you use tags to filter.